ceresdb-example/src/test/java/io/ceresdb/ReadmeTest.java

```diff
@@ -14,29 +14,28 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+// Copyright 2023 CeresDB Project Authors. Licensed under Apache-2.0.
 package io.ceresdb;
 
 import io.ceresdb.models.*;
 import io.ceresdb.options.CeresDBOptions;
+import io.ceresdb.util.StreamWriteBuf;
+
 import org.junit.Assert;
 import org.junit.Ignore;
 import org.junit.Test;
 
 import java.util.List;
 import java.util.concurrent.CompletableFuture;
 import java.util.concurrent.ExecutionException;
-import java.util.stream.Collectors;
+import java.util.stream.Stream;
 
-/**
- * @author kesheng
- */
 public class ReadmeTest {
 
     @Ignore
     @Test
     public void readmeTest() throws ExecutionException, InterruptedException {
-        final CeresDBOptions opts = CeresDBOptions.newBuilder("127.0.0.1", 8831) // CeresDB default grpc port 8831
-                .managementAddress("127.0.0.1", 5000) // CeresDB default http port 3307
+        final CeresDBOptions opts = CeresDBOptions.newBuilder("127.0.0.1", 8831, RouteMode.DIRECT) // CeresDB default grpc port 8831
```

Comment:
Builder should provide a method without `RouteMode` and set the default `RouteMode` to Proxy